.github/scripts/core_checker.py

```diff
@@ -305,11 +305,16 @@
     r'FreeRTOS/Demo/CORTEX_M4F_Infineon_XMC4000_Keil/.*',
     r'FreeRTOS/Demo/AVR_ATMega4809_Atmel_Studio/RTOSDemo/.*',
     r'FreeRTOS/Demo/AVR32_UC3/.*',
+    r'FreeRTOS/Demo/AVR_ATMega4809_IAR/.*',
+    r'FreeRTOS/Demo/AVR_ATMega4809_MPLAB.X/.*',
+    r'FreeRTOS/Demo/AVR_Dx_IAR/.*',
+    r'FreeRTOS/Demo/AVR_Dx_MPLAB.X/.*',
```

Comment:
Isn't this going to skip the copyright check on the source code files we place into these directories?

Reply:
That is correct , the copyright is not added correctly for the files in these folders and I plan to update them in a separate PR, this was done for the CI check.